core/src/css/core.scss

```diff
@@ -7,16 +7,9 @@
 // Ionic Font Family
 // --------------------------------------------------
 
+// TODO(FW-6744): Remove this after adding the ios tokens
 html.ios {
-  --ion-default-font: -apple-system, BlinkMacSystemFont, "Helvetica Neue", "Roboto", sans-serif;
-}
-html.md {
-  --ion-default-font: "Roboto", "Helvetica Neue", sans-serif;
-}
-
-html {
-  --ion-dynamic-font: -apple-system-body;
```

Comment:
This is no longer needed as it is defined by the base tokens: https://github.com/ionic-team/ionic-framework/blob/3fd1d5cab14a866354d5d21fe15e00661ef0f922/core/src/themes/base/default.tokens.ts#L116